Calculator/ViewController.swift

```diff
@@ -73,90 +78,98 @@ class ViewController: UIViewController {
         formatter.maximumIntegerDigits = 8
         return formatter
     }()
-    
-    
-    //Actions Operators
+
+    // Actions Operators
     @IBAction func operatorRestart(_ sender: Any) {
         clear()
     }
+
     @IBAction func operatorDiv(_ sender: UIButton) {
         resultado()
-        operating=true
+        operating = true
         operation = .div
-
     }
+
     @IBAction func operatorMult(_ sender: UIButton) {
         resultado()
-        operating=true
+        operating = true
         operation = .mult
-
     }
+
     @IBAction func operatorRes(_ sender: UIButton) {
         resultado()
-        operating=true
+        operating = true
         operation = .res
-
     }
+
     @IBAction func operatorSum(_ sender: UIButton) {
```

Comment:
You have 5 Actions doing practically the same thing.. Would be better if you only code one Action doing the same, but changing the operation with de button.currentTitle or something like this